### ISSUE TITLE
Regularize text/headings

### ISSUE THREAD

**Issue by WrathfulSpatula** [unitaryfund/metriq-app#660]
Per accessibility benchmark suggestions, styling of text and headings should be regularized. Try to to limit to 2 (or, at most, 3) levels of _directly successive_ header nesting, consistent at each level. Also, try to use a single font (web-safe, readable) and set of styling options for _all_ text besides headings.

**Comment by vprusso** [comment]
I noticed some `h5`, `h4`, etc. tags that I must have missed because I wasn't searching that deep. Once the other outstanding PRs are merged, I'll have another go at this one. @WrathfulSpatula

**Comment by vprusso** [comment]
I'm thinking we might want to allow for lower nesting of headings. 

For one reason, restricting to `<h3>` at most makes the headings look a bit too big I think. For instance:
<img width="1667" alt="Screen Shot 2022-09-03 at 9 25 53 AM" src="https://user-images.githubusercontent.com/1562214/188272471-315aa824-5589-45bb-be30-a519632644e1.png">

vs. 

<img width="1689" alt="Screen Shot 2022-09-03 at 9 25 59 AM" src="https://user-images.githubusercontent.com/1562214/188272490-b58084c3-4ad3-4e1c-90fa-5cc11cc9fcb6.png">

Personally, I think the latter looks better, even though it's an `<h5>` tag. Any thoughts? @WrathfulSpatula

**Comment by WrathfulSpatula** [comment]
@vprusso For accessibility reasons, the correct way to do this is to use site-wide CSS styling on sequential header tag levels. Screen-readers and similar devices have an easier time recognizing and handling the one-at-a-time successive nesting levels of the tags, while how these levels appear can be perfectly arbitrary with CSS. The preferred way to do it, in 2022, is CSS with exactly sequential header tag nesting.

**Comment by WrathfulSpatula** [comment]
@vprusso Let me amend my earlier comment, after having poked around here in the card: the titles on the router views should start at the blue banner header de-emphasis level, +1. Then, I think your suggestion of `<h5>` _already_ was the next de-emphasis level in the hierarchy.

When you look at this again, consider that the de-emphasis level should start from the level after _the blue banner_. But, otherwise, carry on.

**Comment by WrathfulSpatula** [comment]
@vprusso In any case, your suggestion is now _already_ how the header levels are nested, and it's correct, according to me and Lighthouse. We can close the issue, then.
